packages/grafana-ui/src/options/models.gen.ts

```diff
@@ -16,14 +16,14 @@ export interface VizTextDisplayOptions {
  * @public
  */
 export interface OptionsWithLegend {
-  legend: VizLegendOptions;
+  legend?: VizLegendOptions;
```

Comment:
```suggestion
  legend: VizLegendOptions;
```

Reply:
committed this so that we can see the error in CI